core-web/libs/portlets/edit-ema/ui/src/lib/dot-content-compare/dot-content-compare.component.ts

```diff
@@ -27,10 +20,11 @@ export class DotContentCompareComponent {
         }
     }
     @Output() shutdown = new EventEmitter<boolean>();
+    @Output() letMeBringBack = new EventEmitter<{ name: string; args: string[] }>();
     vm$: Observable<DotContentCompareState> = this.store.vm$;
 
     constructor(
-        private store: DotContentCompareStore,
+        public store: DotContentCompareStore,
```

Comment:
why?

Reply:
This component was moved from `dotcms-ui`
store was originally private, but it is used in the template and I got errors
I changed it to avoid those errors